src/flypad-backend/src/FlyPadBackend.cpp

```diff
@@ -89,6 +126,17 @@ bool FlyPadBackend::onUpdate(double deltaTime) {
     lightPresetPtr->onUpdate(deltaTime);
     aircraftPresetPtr->onUpdate(deltaTime);
     pushbackPtr->onUpdate(deltaTime);
+    thirdPartyPtr->onUpdate(simulationData.volumeCOM1, simulationData.volumeCOM2, IVAOData, VPILOTData);
+
+    if(IVAOData) {
+      delete IVAOData;
```

Comment:
If you can use unique_ptr but need to free the memory you could do unique_ptr.reset();

I really don't like seeing new/delete in modern cpp. :)

Reply:
Not sure you should be freeing the memory as I don't think you own it?